src/java.security.jgss/share/classes/sun/security/jgss/krb5/Krb5Context.java

```diff
@@ -641,33 +638,21 @@ public final byte[] initSecContext(InputStream is, int mechTokenSize)
 
                     if (GSSUtil.useSubjectCredsOnly(caller)) {
                         KerberosTicket kerbTicket = null;
-                        try {
-                           // get service ticket from caller's subject
-                           @SuppressWarnings("removal")
-                           var tmp = AccessController.doPrivilegedWithCombiner(
-                                new PrivilegedExceptionAction<KerberosTicket>() {
-                                public KerberosTicket run() throws Exception {
-                                    // XXX to be cleaned
-                                    // highly consider just calling:
-                                    // Subject.getSubject
-                                    // SubjectComber.find
-                                    // instead of Krb5Util.getServiceTicket
-                                    return Krb5Util.getServiceTicket(
-                                        GSSCaller.CALLER_UNKNOWN,
-                                        // since it's useSubjectCredsOnly here,
-                                        // don't worry about the null
-                                        proxyCreds == null ?
-                                            myName.getKrb5PrincipalName().getName():
-                                            proxyCreds.getName().getKrb5PrincipalName().getName(),
-                                        peerName.getKrb5PrincipalName().getName());
-                                }});
-                            kerbTicket = tmp;
-                        } catch (PrivilegedActionException e) {
```

Comment:
The PrivilegedExceptionAction never throws a checked exception, so it was not necessary to retain the code inside the catch block.